.vscode/settings.json

```diff
@@ -0,0 +1,32 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+	// Controls the font size.
```

Comment:
Comments in a JSON file? Does that mean VS Code uses http://json5.org/ ? If so, @aseemk would be amused to know!